python/ambassador/envoy/v3/v3listener.py

```diff
@@ -853,6 +866,12 @@ def finalize_http(self) -> None:
 
             filter_chain: Optional[Dict[str, Any]] = None
 
+            # http/3 is built on quic which has TLS built-in. This means that our UDP Listener will only ever need routes
+            # that match the TLS Filter chain and will diverge from the TCP listener in that it will not support redirect
+            # therefore, we can exclude duplicating the filterchain and routes so hitting this endpoint using non-tls http will fail
+            if (chain.type == "http") & self.isProtocolUDP() & self.http3_enabled:
```

Comment:
`&` is bitwise-and; `and` is boolean-and. This line of code seems to do the right thing, but I'm honestly shocked that it does.

Reply:
my lack of python knowledge biting me. I will get those fixed

Reply:
I have gone ahead and updated to use `and` instead so we should be all good here.